Jenkinsfile

```diff
@@ -59,12 +49,30 @@ def presubmit(gitUtils, bazel) {
     }
     stage('Code Coverage') {
       sh('bin/codecov.sh')
-      withCredentials([string(credentialsId: 'MANAGER_CODECOV_TOKEN', variable: 'CODECOV_TOKEN')]) {
-        sh('curl -s https://codecov.io/bash | bash /dev/stdin -K')
-      }
+      utils.publishCodeCoverage('MANAGER_CODECOV_TOKEN')
     }
     stage('Integration Tests') {
       sh('bin/e2e.sh -t alpha' + gitUtils.GIT_SHA + ' -d')
     }
   }
 }
+
+
+def postsubmit(gitUtils, bazel, utils) {
+  buildNode(gitUtils) {
+    stage('Bazel Tests') {
```

Comment:
Do we need to re-run bazel tests as part of post-submit? My understanding of post-submit hook was that it could not fail so we can't reject the merge at this point. Also, the code coverage step essentially runs an instrumented version of the go tests anyway.